frontend/src/generate-static-html.js

```diff
@@ -30,8 +30,6 @@ export function generateStaticPage({
     <meta name="availableLanguages" content={ available_locales } />
     <meta name="viewport" content="width=device-width" />
 
-    <link rel="localization" href="/static/locales/en-US/app.ftl" />
-    <link rel="localization" href="/static/locales/en-US/experiments.ftl" />
```

Comment:
This was a workaround for #2249 landed in #2287. I managed to track down the root cause. The bug was in the Babel setup of the `compat` build of l20n.js v4. By default Babel doesn't support extending native objects like Errors. L20n uses custom Error subclasses to decide what kind of fallback it should use. Without `instanceof` working correctly due to Babel, L20n never decided to fall back to the next language and instead immediately displayed the id of the translation.